codes/10775_py3_sjskgks.py

```diff
@@ -0,0 +1,23 @@
+import sys
+input = sys.stdin.readline
+
+numOfGate = int(input())
+numOfPlane = int (input())
+possibleDock = [i for i in range(numOfGate+1)]
+numOfDock = 0
+is_full = False
+for _ in range(numOfPlane):
+    try_dock=int(input())
+    if is_full:
+        continue
+    original_try = try_dock
+    while try_dock:
+        if possibleDock[try_dock]==try_dock:
+            numOfDock+=1
+            possibleDock[try_dock]=possibleDock[possibleDock[try_dock-1]]
+            possibleDock[original_try]=possibleDock[try_dock]
+            break
+        try_dock=possibleDock[try_dock]
+    if try_dock==0:
+        is_full=True

```

Comment:
input을 전부 받을 필요는 없습니다. 여기서 break해서 바로 답을 출력하시면 됩니다. is_full은 전부 지워주세요
```suggestion
        break
```